woocommerce/woocommerce-android
# Issue/3030 shipping labels banner

Fixes #3030 by adding a feature announcement banner for shipping labels, if WooCommerce Shipping is available for the store.

#### Changes
- Modified `ProductsWIPNoticeCard` to `FeatureWIPNoticeCard` to make it reusable for all new features in the app.
- Added new feedback url for asking feedback from users for shipping labels.
- Added new banner to Order detail to announce the Shipping Labels feature.

#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/96223828-2e450980-0fac-11eb-8e7c-cfd8f1b17940.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/96223829-300ecd00-0fac-11eb-858d-21008048ea81.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/96223835-31d89080-0fac-11eb-8721-589a0315cbf2.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/96223837-33a25400-0fac-11eb-85fb-d149b132a686.png" width="300"/>

#### Testing
- Click on an order with Shipping Labels. Notice the announcement banner at the top of the screen.
  - Click on the `Give Feedback` button and notice the survey webview. Add your feedback and notice that the your feedback is given and you are redirected to the feedback confirmation screen.
  - Click on the `Dismiss` button and notice that the banner is dismissed and no longer visible.
- Click on an order **without** Shipping Labels. Notice the announcement banner is not displayed.
- Click on the Products tab. Notice the feature announcement banner at the top of the screen. 
  - Click on the `Give Feedback` button and notice the survey webview. Add your feedback and notice that the your feedback is given and you are redirected to the feedback confirmation screen.
  - Click on the `Dismiss` button and notice that the banner is dismissed and no longer visible.


Update release notes:
